keras_retinanet/backend/tensorflow_backend.py

```diff
@@ -17,6 +17,12 @@
 import tensorflow
 
 
+def disable_v2_behavior():
```

Comment:
I know we only support tensorflow, and it's extremely unlikely that another back-end will ever be supported, but: `keras.backend` is meant to abstract away implementation details of backends rather than expose them. That's also why you still need to import tensorflow manually if you want to use things from it directly.

In that light, it may be better not to call this `disable_v2_behavior`, but `disable_tensorflow_v2_behavior`. That still exposes backend details, but not under a generic sounding name.

Not extremely important, but I thought I'd mention it.

Reply:
Yep, not really important, but you're right :+1: